src/arch/arm/kernel/boot.c

```diff
@@ -315,7 +315,27 @@ static BOOT_CODE bool_t try_init_kernel(
     create_frames_of_region_ret_t create_frames_ret;
     create_frames_of_region_ret_t extra_bi_ret;
 
-    /* convert from physical addresses to userland vptrs */
+    /* Convert from physical addresses to userland vptrs with the parameter
+     * pv_offset, which is defined as:
+     *     virt_address + pv_offset = phys_address
+     * The offset is usually a positive value, because the virtual address of
```

Comment:
Well, it is not unusual that physical memory starts from 0.

Reply:
I'm unsure do you want me to change anything here? We have platform where this is the case and platform there this is not the case.